projects/standard-rulesets/src/spectral/index.ts

```diff
@@ -0,0 +1,105 @@
+import { SpectralRule } from '@useoptic/rulesets-base';
+import { Ruleset, Spectral } from '@stoplight/spectral-core';
+import fs from 'fs';
+import fetch from 'node-fetch';
+// @ts-ignore
+import { bundleAndLoadRuleset } from '@stoplight/spectral-ruleset-bundler/with-loader';
+import Ajv from 'ajv';
+import { IChange, IFact, OpenAPIV3, Result } from '@useoptic/openapi-utilities';
+// import fetch from 'node-fetch';
+
+const ajv = new Ajv();
+const configSchema = {
+  type: 'object',
+  properties: {
+    added: {
+      type: 'string',
+    },
+    always: {
+      type: 'array',
+      items: {
+        type: 'string',
+        description: 'URI of spectral ruleset file (file or URL)',
+      },
+    },
+  },
+};
+const validateConfigSchema = ajv.compile(configSchema);
+
+export class SpectralRulesets {
```

Comment:
I think you might need to extend `ExternalRuleBase` here (similar to spectral rule) to get the correct `isInstance` check in the rule runner